CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
 # [Versions](https://mui.com/versions/)
 
+## 6.0.0-beta.3
+
+<!-- generated comparing v6.0.0-beta.2..next -->
+
+_Jul 23, 2024_
+
+A big thanks to the 14 contributors who made this release possible.
```

Comment:
```suggestion
A big thanks to the 14 contributors who made this release possible. Here are some highlights ✨:

- 🚀  New version of the free Dashboard template, now with more components and an improved layout.
```

Maybe we could make this a highlight, since it's the most used template, and it includes many X's components on it.

Reply:
@zanivan I don't see it in the PRs list. Was this already merged before?

Reply:
Found it.